elastic/kibana
# [Infra][Serverless] fix breadcrumb, page template and small layout issues

closes [176602](https://github.com/elastic/kibana/issues/176602)

## Summary

This PR fixes a few small issues that became more evident in the serverless offering

### Serverless

- Breadcrumbs
  <img width="1727" alt="image" src="https://github.com/elastic/kibana/assets/2767137/5808ccd2-8733-406a-8a98-7aedb3e21e8a">
  <img width="1727" alt="image" src="https://github.com/elastic/kibana/assets/2767137/bac99d0c-8146-4946-acc8-7b52133d79fb">
  <img width="1727" alt="image" src="https://github.com/elastic/kibana/assets/2767137/34442e1f-f7ac-425f-9712-ad0a2188cace">


- Asset Details Page Template

| before | after |
| --- | --- |
|<img width="1220" alt="image" src="https://github.com/elastic/kibana/assets/2767137/4bb92ff0-5e27-4ca9-b177-ba2715996648">|<img width="1227" alt="image" src="https://github.com/elastic/kibana/assets/2767137/3d6f1783-01a1-4413-8acf-640b0d6af7f1"> |

The page now uses the `PageTemplate` from `observability-shared`, which is what other pages in observability use.

### Other fixes

 - Spacing between header and unified search in the Hosts View

| before | after |
| --- | --- |
|<img width="884" alt="image" src="https://github.com/elastic/kibana/assets/2767137/49727fc5-0f9b-4ee4-b560-b489c175b1ba">|<img width="885" alt="image" src="https://github.com/elastic/kibana/assets/2767137/858e6930-6210-42a1-8414-bb6e5d60933c"> |

The default spacing is 24px. For some reason, the hosts view had a 12px space between the 2 components.

- Breadcrumb (still works as expected)
  <img width="1714" alt="image" src="https://github.com/elastic/kibana/assets/2767137/ef019d40-2a88-4920-a5dc-ad72a8485536">


### How to test

- Start a serverless Kibana, ES instances and run metricbeat with system module enabled
  - run `yarn es serverless --projectType=oblt` and `yarn serverless-oblt`
  - Navigate to Inventory pages and APM Settings
  - Check the changes described above
- Start a stateful Kibana instance
  - Navigate to inventory pages and APM Settings
  - Check the changes described above
